src/components/posts/MainCommentComponent.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useState } from 'react';
+import CommentForm from './PostCommentListItemForm';
+import PreviewComment from './PreviewComment';
+import CommentList from './PostCommentList';
+import ShowAll from './ShowAllToggle';
+
+function PostComments({ onCommentAdded, post }) {
+  const [showAll, setShowAll] = useState(false);
+
+  const toggleCommentsList = () => {
+    setShowAll((toggle) => !toggle);
+  };
+
+  return (
+    <div className='comments-section'>
+      {post.id && <CommentForm post={post} onCommentAdded={onCommentAdded} />}
```

Comment:
Ah sorry, didn't catch this conditional render last review! 

Could you also split this one out please?